### PR TITLE
Fix path in cri-containerd e2e.

### PR DESCRIPTION
**Pull request by Random-Liu**
From the test log, it seems that current directory is `/workspace`.
Use relative path now.

### CODE REVIEW

**Comment by krzyzacy** [comment]
the file dir structure looks like:
```
- workspace
  - kubernetes
    - kubernetes binary stuff (* cwd)
- go
  -src
    -github.com
      -cri-containerd repo
```

**Comment by Random-Liu** [comment]
@krzyzacy 
```
I1107 02:42:32.919] Checkout: /workspace/github.com/kubernetes-incubator/cri-containerd master
```
From the log, it is not.
https://storage.googleapis.com/kubernetes-jenkins/logs/ci-cri-containerd-e2e-ubuntu-gce/2/build-log.txt

**Comment by krzyzacy** [comment]
@Random-Liu sorry I thought you have something like https://github.com/Random-Liu/test-infra/blob/464bf02672dbed2725d0f68288c733930f3e7d64/prow/config.yaml#L59 set

File: jobs/env/ci-cri-containerd-e2e-gce.env [hunk 1]
```diff
@@ -10,8 +10,8 @@ TEST_ETCD_VERSION=2.2.1
 ENABLE_POD_SECURITY_POLICY=true
 
 # Envs for cri-containerd.
-KUBE_MASTER_EXTRA_METADATA=user-data=/go/src/github.com/kubernetes-incubator/cri-containerd/test/e2e/master.yaml,cri-containerd-configure-sh=/go/src/github.com/kubernetes-incubator/cri-containerd/test/configure.sh
-KUBE_NODE_EXTRA_METADATA=user-data=/go/src/github.com/kubernetes-incubator/cri-containerd/test/e2e/node.yaml,cri-containerd-configure-sh=/go/src/github.com/kubernetes-incubator/cri-containerd/test/configure.sh
+KUBE_MASTER_EXTRA_METADATA=user-data=github.com/kubernetes-incubator/cri-containerd/test/e2e/master.yaml,cri-containerd-configure-sh=github.com/kubernetes-incubator/cri-containerd/test/configure.sh
```
Comment (krzyzacy) [inline jobs/env/ci-cri-containerd-e2e-gce.env]: then the relative path will be `../github.com/...`?

Comment (Random-Liu) [reply jobs/env/ci-cri-containerd-e2e-gce.env]: I'm completely confused. :p 
So I think the safest way is to specify `--root=/go/src`, and use abusolute path here?

Comment (Random-Liu) [reply jobs/env/ci-cri-containerd-e2e-gce.env]: @krzyzacy Changed to absolute path `/workspace/github.com/XXX`.

This should work now.

**Pull request merged**